### ISSUE TITLE
How to write a message to user and how to get new messages sent by user?

### ISSUE THREAD

**Issue by taseppa** [charliewolf/pynder#130]
Couldn't find anything in frontpage or examples related to this. Also looking at Tinder API documentation https://gist.github.com/rtt/10403467 i'm not sure how new messages are supposed to be received anyway.

**Comment by taseppa** [comment]
Digging around the code api.py seems to have updates function that should do what I want and user has message function. I think this updates should also be added to session.py

**Comment by fishtanker** [comment]
The message function is in this python file. https://github.com/charliewolf/pynder/blob/master/pynder/api.py

**Comment by charliewolf** [comment]
You shouldn't be using that function in `api.py` directly, just call `match.message(...)` where `match` is an instance of `Match`

**Comment by taseppa** [comment]
That's correct. But currently there seems to be missing support for updates request.

 Also regarding messaging, let's say you know only user's tinder id, how can you send a message in this case?

**Comment by charliewolf** [comment]
So you're saying for a user that isn't a match?

**Comment by taseppa** [comment]
The user is a match of course. Basically I think what is missing is how to get a match based only on ID without having to get all matches.
